extra-games/qzdl/autobuild/patch

```
@@ -0,0 +1,3 @@
+abinfo 'Patching CMakeList.txt to install binaries automatically...'
+sed -e '53a install(TARGETS zdl DESTINATION ${CMAKE_INSTALL_BINDIR})' \
```

Use a patch instead, sed is unreliable and can silently fail.